exatomic/adf/output.py

```diff
@@ -185,7 +185,7 @@ def parse_basis_set_order(self):
                         data['n'].append(n)
                         data['r'].append(r)
         data['set'] = data.pop('seht')
-        data['frame'] = 0
+        data['frame'] = [0]*len(data['set'])
```

Comment:
Can you explain this change? Doing `data["frame"] = 0` sets the entire series "frame" to 0. The change does the same exact thing that pandas does automatically.

Reply:
I remember at the time I was having an issue because it initializes everything as a dictionary from collections and it wasn't doing that automatically. A couple lines down it converts the dictionary into a pandas dataframe.